iox_query_influxql/src/plan/planner_time_range_expression.rs

```diff
@@ -103,7 +104,7 @@ fn reduce_expr(expr: &Expr, tz: Option<chrono_tz::Tz>) -> ExprResult {
     match expr {
         Expr::Binary(v) => reduce_binary_expr(v, tz).map_err(map_expr_err(expr)),
         Expr::Call (Call { name, .. }) => {
-            if !name.eq_ignore_ascii_case("now") {
+            if !is_now_function(&name.to_lowercase()) {
```

Comment:
```suggestion
            if !is_now_function(&name) {
```

Same here - sorry I omitted this when I changed the behaviour of the call expression parsing.